### DISCUSSION TITLE
Did something change with the catppuccin theme?

### DISCUSSION THREAD

**Discussion by DKulan** [LazyVim/LazyVim#4230] [Q&A] [answered]
Was there an update that I missed to the Catppuccin theme? it looks drastically different.

<img width="1779" alt="Screenshot 2024-07-30 at 2 08 43 PM" src="https://github.com/user-attachments/assets/19fcfc0b-4843-4b12-af58-3ecf416d2886">

**Comment by Krystian-KF** [comment]
Do other themes look funky? It looks to me you are running nvim through tmux. 
If so, add the following to your .tmux.conf:

```
set -g default-terminal "${TERM}"
set-option -ga terminal-overrides ",xterm-256color:Tc"
```

**Reply by DKulan** [reply]
thanks @Krystian-KF, I tried that and reloaded but still seeing the same. I don't think I messed around with my tmux/nvim config in awhile but I did update lazyvim plugins through the menu which caused the colors to change.

**Reply by Krystian-KF** [reply]
Interesting, I am using Catppuccin theme myself as well and have not experience any issues recently (apart from the tmux one, way back in the past when I started using it and didn't know how to configure it).
The palette looks like a terminal only support 16 colours. The only other thing I can think of is to try a different one (terminal emulator) and/or putting in your shell config: `export TERM=xterm-256color`

**Reply by DKulan** [reply, marked as answer]
got it! I am using Alacritty and had to set the correct color support in my alacritty .toml config file:

```
[env]
TERM = "xterm-256color"
```

weird that it used to work before without me having to set that.
